File: Arithmetic_LCM.cpp
// Arithmetic.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include "LCM.h"

#include <iostream>
#include <sstream>
#include <vector>

using namespace std;

int main() {
    try {
        cout << "\n    Welcome to our demonstration sequels\n";
        cout << "Hope you enjoy (and follow) the lessons.\n\n";

        vector<unsigned> set;

        /*
         * Find L.C.M
         */
        set = {2, 5, 7};

        stringstream aux;
        string render_result = "The LCM of ";
        for (unsigned number : set) {
            aux.str("");
            aux << number;
            render_result += aux.str() + "; ";
        }
        render_result += " is:  ";

        LCM lcm(set);
        cout << "\n\n" << render_result << lcm.getLCM() << "\n";

    } catch (exception& e) {
        cout << "\n" << e.what() << "\n";
    }
    return 0;
}

File: Arithmetic_LCM.java
package arithmetic;

import java.util.ArrayList;
import java.util.List;

public class Arithmetic {

    public static void main(String[] args) {
        System.out.println("Welcome to our demonstration sequels");
        System.out.println("Hope you enjoy (and follow) the lessons.");
        System.out.println("");

        /*
        * Find LCM.
         */
        String render_result = "The LCM of ";
        LCM lcm;
        List<Integer> group;

        group = new ArrayList<>();
        group.add(40);
        group.add(50);
        group.add(60);

        for (int number : group) {
            render_result += number + "; ";
        }
        render_result += "is:  ";

        lcm = new LCM(group);
        System.out.println(render_result + lcm.getLCM());
    }
}
